tests/token_provision.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Library     SSHLibrary    timeout=90 seconds
+Library     Process
+Library     OperatingSystem
+Library     String
+Library     RequestsLibrary
+Library     Collections
+
+Suite Setup       Run Keyword    Open Server
+Suite Teardown    Run Keyword    SSHLibrary.Close All Connections
+
+Resource    ../variables.robot
+Resource    ../keywords.robot
+
+*** Test Cases ***
+
+TOP001.001 Token provisioning
+    [Documentation]    Checks that API command admin/token_provision works
+    ...                correctly.
+    [Teardown]    Run Keyword If Test Failed    Set Suite Variable    ${prev_test_status}    FAIL
+    Run    python3 provisioning/cc_cbor.py provisioning/chain.pem provisioning/tmp/fobnail.cbor
+    Run    coap-client -t application/cbor -m post -f provisioning/tmp/fobnail.cbor coap://${fobnail_ip}/api/v1/admin/token_provision > provisioning/tmp/fobnail.csr
```

Comment:
This will silently fail to create `provisioning/tmp/fobnail.csr` if `provisioning/tmp` does not exist

Reply:
It is possible to add an empty folder?